packages/core/src/app/shipping/hasSelectedShippingOptions.ts

```diff
@@ -9,12 +9,14 @@ export default function hasSelectedShippingOptions(consignments: Consignment[]):
     return every(
         consignments,
         (consignment) =>
-            consignment.selectedShippingOption &&
-            consignment.selectedShippingOption.id &&
-            // Selected option is available
-            consignment.availableShippingOptions &&
-            consignment.availableShippingOptions.filter(
-                ({ id }) => id === consignment.selectedShippingOption?.id,
-            ).length,
+            (consignment.selectedShippingOption &&
+                consignment.selectedShippingOption.id &&
+                // Selected option is available
+                consignment.availableShippingOptions &&
+                consignment.availableShippingOptions.filter(
+                    ({ id }) => id === consignment.selectedShippingOption?.id,
+                ).length) ||
+            (consignment.selectedShippingOption?.type === 'custom' &&
+                consignment.selectedShippingOption.cost >= 0),
```

Comment:
Don't think we need the cost comparison.

Reply:
Thanks for the feedback.
All comments have been addressed in the last commit, https://github.com/bigcommerce/checkout-jas/pull/1900/commits/b83b798c9769fe3c9ec5e5d87cc5270312bda14a.